src/commands/games/fivem.js

```diff
@@ -14,40 +14,43 @@ module.exports = class FiveM extends Command {
+        })
 
-    if (server) {
-      embed.setAuthor('FiveM', 'https://i.imgur.com/u5wzB9A.png')
-        .setTitle(`${server.hostname.replace(/\|/g, '\\|').substring(0, 253)}...`)
-        .setURL(`https://servers.fivem.net/servers/detail/${id}`)
-        .setThumbnail(`https://servers-live.fivem.net/servers/icon/${id}/${server.iconVersion}.png`)
-        .addField(t('commands:fivem.address'), `\`cfx.re/join/${id}\``, true)
-        .addField(t('commands:fivem.players'), `${server.players.length}/${server.sv_maxclients}`, true)
-        .addField(t('commands:fivem.map'), server.mapname || t('commands:fivem.noMap'), true)
-        .addField(t('commands:fivem.gameMode'), server.gametype || t('commands:fivem.noGameMode'), true)
-        .addField(t('commands:fivem.gameName'), server.vars.gamename || t('commands:fivem.noGameName'), true)
+      if (server) {
+        embed.setAuthor('FiveM', 'https://i.imgur.com/hIurCNU.png')
+          .setTitle(`${server.hostname.replace(/\|/g, '\\|').replace(testregex, '').substring(0, 253)}...`)
+          .setURL(`https://servers.fivem.net/servers/detail/${id}`)
+          .setThumbnail(`https://servers-live.fivem.net/servers/icon/${id}/${server.iconVersion}.png`)
+          .addField(t('commands:fivem.address'), `\`cfx.re/join/${id}\``, true)
+          .addField(t('commands:fivem.players'), `${server.players.length}/${server.sv_maxclients}`, true)
+          .setColor(Constants.FIVEM_COLOR)
+          .addField(t('commands:fivem.map'), server.mapname || t('commands:fivem.noMap'), true)
+          .addField(t('commands:fivem.gameMode'), server.gametype || t('commands:fivem.noGameMode'), true)
+          .addField(t('commands:fivem.gameName'), server.vars.gamename || t('commands:fivem.noGameName'), true)
 
-      if (server.vars.banner_detail || server.vars.banner_connecting) {
-        embed.setImage(server.vars.banner_detail ? server.vars.banner_detail : (server.vars.banner_connecting ? server.vars.banner_connecting : null))
-      }
+        const allowedExtensions = ['png', 'jpg', 'gif']
+        if (allowedExtensions.some(e => server.vars.banner_detail.endsWith(e)) || allowedExtensions.some(e => server.vars.banner_connecting.endsWith(e))) {
+          embed.setImage(server.vars.banner_detail ? server.vars.banner_detail : (server.vars.banner_connecting ? server.vars.banner_connecting : null))
+        }
 
-      channel.send(embed).then(channel.stopTyping())
+        channel.send(embed).then(() => channel.stopTyping())
+      }
+    } catch {
+      throw new CommandError(`${t('commands:fivem.serverUnreachableTitle')} ${t('commands:fivem.serverUnreachableDescription')}`)
     }
   }
 
-  getAllServers () {
-    return new Promise(function (resolve, reject) {
-      fetch('http://servers-live.fivem.net/api/servers/')
-        .then(res => resolve(res.json()))
-        .catch(error => reject(error))
-    })
+  async getAllServers () {

```

Comment:
```suggestion
  getAllServers () {
```

Reply:
ae @metehus aprende com ele a fazer sugestão 😎

Reply:
to no celular

Reply:
aporra